.github/workflows/cherry-pick-to-trunk.yml

```
@@ -0,0 +1,264 @@
+        with:
+          slack-bot-user-oauth-access-token: ${{ secrets.CODE_FREEZE_BOT_TOKEN }}
+          slack-optional-unfurl_links: false
+          slack-channel: ${{ secrets.WOO_RELEASE_SLACK_NOTIFICATION_CHANNEL }}
+          slack-text: |
+            :info: *Cherry pick <${{ env.CHERRY_PICK_PR_URL }}|created> (${{ env.BASE_BRANCH }} → `trunk`)* Please ensure <${{ env.CHERRY_PICK_PR_URL }}|this cherry pick> is merged before releasing. Source PR: _<${{ env.SOURCE_PR_URL }}|#${{ env.SOURCE_PR_NUMBER }} - ${{ env.SOURCE_PR_TITLE }}>_.
+            ${{ env.ADD_CP_TO_FROZEN_NAG == 'true' && format('\\n\\nNOTE: This PR might need to be cherry-picked to the frozen release ({0}).', env.NEXT_BRANCH) || '' }}
+
+  handle-failures:
+    needs: [prepare, cherry-pick, frozen-release-check]
+    runs-on: ubuntu-latest
+    if: ${{ always() && needs.cherry-pick.ouputs.status == 'failure' }}
```

actionlint (1.7.7)

property "ouputs" is not defined in object type {outputs: {cherry_pick_pr_number: string; error_message: string; status: string}; result: string}

(expression)